sympy/solvers/ode.py

```diff
@@ -4094,11 +4094,14 @@ def unreplace(eq, var):
     var = func.args[0]
     subs_eqn = replace(eq, var)
     try:
-        solns = solve(subs_eqn, func)
+        # turn off simplification to protect Integrals that have
+        # _t instead of fx in them and would otherwise factor
+        # as t_*Integral(1, x)
```

Comment:
The other option would be to use an `intx` analogous to `diffx` that would not be recognized as an Integral needing to be factored, thus keeping the temporary `t_` in place.